tests/test_emr/test_utils.py

```diff
@@ -47,3 +50,125 @@ def test_release_label_comparisons():
         "emr-5.60.2",
     ]
     assert releases_sorted == expected
+
+
+@pytest.mark.skipif(
+    settings.TEST_SERVER_MODE, reason="Can't modify backend directly in server mode."
+)
+class TestEmrSecurityGroupManager(object):
```

Comment:
Same here with this test, I guess - that can be moved to `test_emr_util_integration.py`